aws/resource_aws_msk_cluster_test.go

```diff
@@ -103,8 +104,9 @@ func TestAccAWSMskCluster_basic(t *testing.T) {
 				ImportState:       true,
 				ImportStateVerify: true,
 				ImportStateVerifyIgnore: []string{
-					"bootstrap_brokers",     // API may mutate ordering and selection of brokers to return
-					"bootstrap_brokers_tls", // API may mutate ordering and selection of brokers to return
+					"bootstrap_brokers",              // API may mutate ordering and selection of brokers to return
+					"bootstrap_brokers_tls",          // API may mutate ordering and selection of brokers to return
+					"broker_client_vpc_ip_addresses", // API may mutate ordering and selection of brokers to return
```

Comment:
this shouldn't be an issue for a set

Reply:
Thanks, removed all cases where broker_client_vpc_ip_address was included under ImportStateVerifyIgnore in https://github.com/terraform-providers/terraform-provider-aws/pull/12821/commits/f48c1890c4d5a9c972c0a6166970a4ffc1f73272